include/deal.II/base/vectorization.h

```diff
@@ -2252,6 +2314,14 @@ class VectorizedArray<double, 4>
     : VectorizedArrayBase<VectorizedArray<double, 4>, 4>(list)
   {}
 
+  /**
+   * Construct an array with the data field.
+   */
+  VectorizedArray(__m256d const &x)
```

Comment:
Could you please remove this constructor and all other constructors with `_m256`/`_m128` variables? The `data` fields of the respective type are `public` already, so I would prefer to assign the `VectorizedArray::data` field directly instead of making the API broader. My main concern is that we can not use constructors with say `_m256` in a portable way, so it might mislead one to write code that only works on a specific platform.

Reply:
Sure!